### PR TITLE
Fix broken link for qrel.dev.small.tsv

### PR DESCRIPTION
**Pull request by larryli1999**
Update the link to download qrels.dev.small.tsv

### CODE REVIEW

File: docs/experiments-monot5-tpu.md
Comment (rodrigonogueira4) [inline docs/experiments-monot5-tpu.md:74]: remove `?dl=1`?

**Review by rodrigonogueira4 — APPROVED** [review]
LGTM. Just a minor comment.

**Comment by lintool** [comment]
hey @larryli1999 @ronakice are these the same ones here? https://github.com/castorini/anserini/tree/master/src/main/resources/topics-and-qrels

If so, fetch from there. That's stable.

**Comment by larryli1999** [comment]
I have updated the github links as well as the corresponding file names in the documentation.

**Comment by ronakice** [comment]
Okay LGTM after the new edits, merging! Thanks :)

**Pull request merged**
